chia/protocols/harvester_protocol.py

```diff
@@ -51,6 +55,21 @@ class NewProofOfSpace(Streamable):
     plot_identifier: str
     proof: ProofOfSpace
     signage_point_index: uint8
+    include_source_signature_data: Optional[bool]
```

Comment:
same here, unless I'm missing something subtle, I would expect this to just be a `bool`